model/Image.php

```diff
@@ -70,7 +70,7 @@ class Image extends File implements Flushable {
 	 * @config
 	 * @var bool Force all images to resample in all cases
 	 */
-	private static $force_resample = false;
+	private static $force_resample = true;
```

Comment:
@tractorcow I included a [filesize check](https://github.com/silverstripe/silverstripe-framework/pull/4229/files#r31054055) so that the resampled one would only be served up if it's smaller than the original, do you think that won't be enough to prevent deterioration?